e2e/rbd_helper.go

```diff
@@ -314,6 +314,38 @@ func deleteBackingRBDImage(f *framework.Framework, pvc *v1.PersistentVolumeClaim
 	return err
 }
 
+func backingRBDImageHasExtents(f *framework.Framework, pvc *v1.PersistentVolumeClaim) (bool, error) {
+	imageData, err := getImageInfoFromPVC(pvc.Namespace, pvc.Name, f)
+	if err != nil {
+		return false, err
+	}
+
+	cmd := fmt.Sprintf("rbd diff %s %s", rbdOptions(defaultRBDPool), imageData.imageName)
+	stdout, stderr, err := execCommandInToolBoxPod(f, cmd, rookNamespace)
+	if err != nil {
+		return false, err
+	}
+	if stderr != "" {
+		return false, fmt.Errorf("command %q returned error output: %s", cmd, stderr)
+	}
+	if stdout == "" {
+		return false, nil
+	}
+
+	return true, nil
+}
+
+func sparifyBackingRBDImage(f *framework.Framework, pvc *v1.PersistentVolumeClaim) error {
```

Comment:
are we just making sure we are able to run sparsify on an image. and also is this command available on all ceph versions?

Reply:
yeah, verify that a thick-provisioned image (with zero's as data) can be sparsified. This is not something that would be done on a deployment, but it validates that the image only contains zero's.

Reply:
Added comment to explain the need/usage of the function.

Reply:
`rbd sparsify` is available in only the newer ceph version right?

Reply:
It was made available with Nautilus. That is the oldest Ceph release that is still maintained.